src/parameters.f90

```diff
@@ -635,6 +638,10 @@ subroutine parameter_defaults()
   cpg = .false.
   idir_stream = 1
 
+  !! Pipe
+  ra  = 0.5 !inner radius
+  rao = 0.6 !outer radius
```

Comment:
I think `ra` already existed. I'm not sure it is safe to set defaults like this. Better perhaps to set an invalid default (e.g. `rao = -1.0`) and check this inside the case-specific code, killing the run if it hasn't been specified a valid value.

Reply:
In 45f858ff63db22be5cf211f312554320d024974a , I've introduced a new variable for the pipe inner radius (`rai`) while imposing negative values for `rai` and `rao` together with a safety check, as suggested.